collect_app/src/main/res/values/dark_theme.xml

```diff
@@ -71,4 +71,8 @@
     <style name="Widget.Collect.SearchView.Dark" parent="Widget.AppCompat.SearchView">
         <item name="searchHintIcon">@null</item>
     </style>
+
+    <style name="Theme.Collect.Dark.Dialog" parent="android:Theme.Material.Dialog">
+        <item name="android:colorAccent">#57C4C2</item>
```

Comment:
I think `colorAccent` values should be our theme's `colorSecondary` attribute both here and in the `light_theme`.

Reply:
Oh and it looks like we should correct this for `Dialog.Alert. themes as well.

Reply:
Done.